homeassistant/components/google/__init__.py

```diff
@@ -302,11 +322,20 @@ def _add_event(call):
 
         event = {
             "summary": call.data[EVENT_SUMMARY],
+            "status": call.data[EVENT_STATUS],
             "description": call.data[EVENT_DESCRIPTION],
             "start": start,
             "end": end,
+            "location": call.data[EVENT_LOCATION],
+            "recurrence": call.data[EVENT_RECURRENCE],
+            "transparency": call.data[EVENT_TRANSPARENCY],
+            "visibility": call.data[EVENT_VISIBILITY],
+        }
+        service_data = {
+            "calendarId": call.data[EVENT_CALENDAR_ID],
+            "send_updates": call.data[EVENT_SEND_UPDATES],
```

Comment:
I was double checking this against https://developers.google.com/calendar/v3/reference/events/insert and was curious if "send_updates" needs to instead be "sendUpdates"? I could believe the API does the translation, but wanted to make sure.

Reply:
Your right. I would like to have `send_updates` be used by the user. Google uses `sendUpdates` instead.